docs/dev/search-integration.rst

```diff
@@ -1,7 +1,7 @@
 Server Side Search Integration
 ==============================
 
-Read the Docs provides :doc:`server side search (SSS) <rtd:server-side-search>`
+Read the Docs provides :doc:`server side search (SSS) <server-side-search>`
```

Comment:
The user article has been refactored to a folder structure now:

https://docs.readthedocs.io/en/stable/server-side-search/index.html

However, the above link is already in a dev article, so perhaps it should always have been for its dev ditto.